src/platform/K32W/CHIPDevicePlatformConfig.h

```diff
@@ -77,11 +77,11 @@
 
 // ========== Platform-specific Configuration Overrides =========
 #ifndef CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE
-#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE 9216
+#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE (5 * 1024)
```

Comment:
I had to increase this task to 6 k on master on EFR32 platform just yesterday due to a stack overflow just after CASE session completes. Did you validate that if 5k is enough on ToT master for NXP platform.

Reply:
Yes, it looks that 5K are enough for DK6-K32W061 board. I tested this using the watermark feature from FreeRTOS.